src/Task.cc

```diff
@@ -1345,13 +1366,35 @@ void Task::flush_regs() {
   if (registers_dirty) {
     LOG(debug) << "Flushing registers for tid " << tid << " " << registers;
     auto ptrace_regs = registers.get_ptrace();
+#if defined(__i386__) || defined(__x86_64)
```

Comment:
It seems your `__x86_64__` got its right arm torn off in some sort of terrible industrial accident.

Reply:
Indeed. No idea why this ever passed tests